projects/js-packages/publicize-components/src/hooks/use-image-generator-config/index.js

```diff
@@ -46,14 +46,12 @@ export default function useImageGeneratorConfig() {
 		[ currentOptions, editPost, postSettings ]
 	);
 
-	const getCurrentSettings = sigSettings => {
-		return {
-			isEnabled: sigSettings?.enabled ?? ! isPostPublished,
-			customText: sigSettings?.custom_text ?? null,
-			imageType: sigSettings?.image_type ?? null,
-			imageId: sigSettings?.image_id ?? null,
-		};
-	};
+	const getCurrentSettings = sigSettings => ( {
+		isEnabled: sigSettings?.enabled ?? ! isPostPublished,
+		customText: sigSettings?.custom_text ?? null,
+		imageType: sigSettings?.image_type ?? null,
+		imageId: sigSettings?.image_id ?? null,
+	} );
```

Comment:
I've just realised that we could define this function outside of the hook, but maybe that's an optimisation too far!